pkg/oracle/oracle.go

```diff
@@ -80,6 +82,9 @@ const (
 
 	// defaultRequestTimeout is default request timeout.
 	defaultRequestTimeout = time.Second * 5
+
+	// defaultMaxTaskTimeout is default timeout for the failed request to be reprocessed.
+	defaultMaxTaskTimeout = time.Minute * 3
```

Comment:
Too low. It should be at least one hour (for MaxTaskTimeout, not RefreshInterval).

Reply:
Fixed.